src/Instrumentation/PDO/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "open-telemetry/opentelemetry-auto-pdo",
+  "description": "OpenTelemetry auto-instrumentation for PDO",
+  "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "pdo", "instrumentation"],
+  "type": "library",
+  "homepage": "https://opentelemetry.io/docs/php",
+  "readme": "./README.md",
+  "license": "Apache-2.0",
+  "minimum-stability": "dev",
+  "require": {
+    "php": "^8.0",
+    "ext-otel_instrumentation": "*",
+    "open-telemetry/api": "^1",
+    "nyholm/psr7": "^1",
```

Comment:
I think nyholm/* is not required for this instrumentation

Reply:
Yes, right. Will remove it.